packages/crypto-wallet-core/test/validation.ts

```diff
@@ -17,9 +17,10 @@ describe('Address Validation', () => {
 
   // Uri
   const btcUri = 'bitcoin:1NuKwkDtCymgA1FNLUBaUWLD8s4kdKWvgn';
-  const bchUri = 'bitcoincash:qr8uujscckc56ancdkmqnyyl2rx6pnp24gmdfrf8qd';
+  const bchUri = 'bitcoincash:pp8skudq3x5hzw8ew7vzsw8tn4k8wxsqsv0lt0mf3g';
+  const ethUri = 'ethereum:0x37d7B3bBD88EFdE6a93cF74D2F5b0385D3E3B08A?value=123?gasPrice=123?gas=123?gasLimit=123'
```

Comment:
should the URI be:

ethereum:0x37d7B3bBD88EFdE6a93cF74D2F5b0385D3E3B08A?value=123&gasPrice=123&gas=123&gasLimit=123

could you also add a naked one, like:
ethereum:0x37d7B3bBD88EFdE6a93cF74D2F5b0385D3E3B08A

Reply:
added